pom.xml

```diff
@@ -544,7 +544,7 @@ under the License.
 				<artifactId>jackson-bom</artifactId>
 				<type>pom</type>
 				<scope>import</scope>
-				<version>2.13.0</version>
+				<version>2.13.2.20220328</version>
```

Comment:
As mentioned here https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.13#micro-patches for jackson-databind 2.13.2.2